docs/user-guide/release-notes/2.11.x.rst

```diff
@@ -13,4 +13,7 @@ New Features
   since the last sync.
 * The yum_importer can sync a distribution unit even when no yum metadata is present in the repo.
   This is useful for kickstart trees related to Project Atomic.
-* Added support for publishing ISO repositories to custom relative paths.
\ No newline at end of file
+* Added support for publishing ISO repositories to custom relative paths.
+* RPMs are now being published into new folder structure e.g. "Packages/d/dnf.rpm", this applies
```

Comment:
We add new features to Y releases only, so please move this to 2.12 release notes. For the same reason PR should be opened against master branch as @mhrivnak  mentioned earlier.
I am leaving for the native speakers to check the wording.